docs/dxp/7.2.x/en/platform/app-builder/app-builder.md

```diff
@@ -0,0 +1,54 @@
+# Low-Code Application Development with App Builder
+
+Writing applications for Liferay DXP has never been easier. Introduced with Liferay DXP 7.3, App Builder provides a low-code application development solution. There are four concepts at the root of App Builder apps:
+
+- [Custom Objects](#custom-objects)
```

Comment:
On DXP 7.3 EE it's very likely to already have the Native Objects as well, but because the DXP 7.3 CE GA1 doesn't have Native objects yet, I am not sure if we already need to mention this concept here as well.